enclave-runtime/src/initialization.rs

```diff
@@ -15,6 +15,13 @@
 
 */
 
+use base58::ToBase58;
+use codec::Encode;
+use log::*;
+use primitive_types::H256;
+use sp_core::crypto::Pair;
+use std::{string::String, sync::Arc};
+
```

Comment:
These imports are not grouped and sorted with the ones below, because there is a space in between. Group all the `use` together and let the formatter do the ordering.

Reply:
ups!